src/Explorer/Panes/ExecuteSprocParamsPanel/InputParameter.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  Dropdown,
+  IDropdownOption,
+  IDropdownStyles,
+  IImageProps,
+  Image,
+  Label,
+  Stack,
+  TextField,
+} from "office-ui-fabric-react";
+import React, { FunctionComponent } from "react";
+import Add_property from "../../../../images/Add-property.svg";
+import Entity_cancel from "../../../../images/Entity_cancel.svg";
```

Comment:
Can we use camelcase and suffix with Icon ? (`AddPropertyIcon` and `EntityCancelIcon`)

Reply:
Done. Thank you.